mimic/rest/nova_api.py

```diff
@@ -219,8 +219,16 @@ def create_server(self, request, tenant_id):
             request.setResponseCode(400)
             return json.dumps(invalid_resource("Invalid JSON request body"))
 
-        return (self._region_collection_for_tenant(tenant_id)
-                .request_creation(request, content, self.url))
+        try:
+            creation = (self._region_collection_for_tenant(tenant_id)
+                        .request_creation(request, content, self.url))
+        except ValueError:
+            request.setResponseCode(400)
+            return json.dumps(
```

Comment:
Filed:  https://github.com/rackerlabs/mimic/issues/186

Sorry, realized I wasn't super clear, I just meant change "OS-DCF:diskConfig is neither AUTO nor MANUAL" to "OS-DCF:diskConfig must be either 'MANUAL' or 'AUTO'.".

Someone else can come along and swap out the function later.  :)